src/components/editor/faultTree/canvas/EditorCanvas.tsx

```diff
@@ -216,18 +217,24 @@ const EditorCanvas = ({
   };
 
   useEffect(() => {
-    if (container && rootEvent) {
-      container.removeCells(container.getCells());
-
-      const listOfPaths = [];
-
-      if (targets.length > 0 && showPath) {
-        targets.forEach((target) => listOfPaths.push(findNodeByIri(rootEvent, target)));
+    const renderAndLayout = async () => {
```

Comment:
I am not sure to understand why we need it asynchronous now, and not before?

Reply:
I changed it to be able make this function as async. Only for loading: 
<img width="657" alt="Screenshot 2024-04-11 at 5 47 29 PM" src="https://github.com/kbss-cvut/fta-fmea-ui/assets/49252228/99eb44c7-af4a-46ed-8296-cd9a24b1357d">